src/megaclient.cpp

```diff
@@ -3709,12 +3709,19 @@ void MegaClient::resumeResumableSyncs()
     {
         return;
     }
-    for (const auto& config : syncConfigs->all())
+    for (auto config : syncConfigs->all())
     {
         if (!config.isResumable())
         {
             continue;
         }
+        if (!nodebyhandle(config.getRemoteNode()))
+        {
+            // remote node gone
+            config.setResumable(false);
+            syncConfigs->insert(config);
+            continue;
+        }
```

Comment:
Below this comment, if the sync fails to be resumed (else of `if (e == 0)`), shouldn't we update the status / resumable-flag of the sync (and notify the app accordingly)? That way, the app will learn the sync was not resumed by "reason"

Reply:
Yes, this is done now.